apps/mux/src/config.tsx

```diff
@@ -255,6 +347,45 @@ class Config extends React.Component<ConfigProps, IState> {
             </Form>
           </Typography>
           <hr className="config-splitter" />
+          <Form spacing="default">
+            <Heading>Advanced: Signed URLs</Heading>
+            <Note noteType="warning" title="This is an advanced feature">
+              If you want to support signed urls you must read and understand{' '}
+              <TextLink
+                href="https://docs.mux.com/docs/headless-cms-contentful#advanced-signed-urls"
+                target="_blank"
```

Comment:
Could you please add `rel="noopener noreferrer"`?

Reply:
done!